src/master-playlist-controller.js

```diff
@@ -369,10 +327,20 @@ export default class MasterPlaylistController extends videojs.EventTarget {
     }
   }
 
-  useAudio(label) {
-    let audioEntry;
-    let newAudioPlaylistLoader;
-    let mediaGroupName = this.masterPlaylistLoader_.media().attributes.AUDIO;
+  useAudio() {
+    let media = this.masterPlaylistLoader_.media();
+    let master = this.masterPlaylistLoader_.master;
+
+    if (!media || !media.attributes || !media.attributes.AUDIO ||
+        !master.mediaGroups || !master.mediaGroups.AUDIO) {
+      return;
+    }
+    let mediaGroupName = media.attributes.AUDIO;
+
+    if (!master.mediaGroups.AUDIO[mediaGroupName]) {
+      return;
+    }
+    let audioEntries = master.mediaGroups.AUDIO[mediaGroupName];
```

Comment:
from what I understood we only did that when we used var to declare all variables at the top, comma separated. I don't think we need them all to be at the top with let.